engelsystem/src/test/kotlin/info/metadude/android/eventfahrplan/engelsystem/extensions/CallExtensionsTest.kt

```diff
@@ -0,0 +1,176 @@
+package info.metadude.android.eventfahrplan.engelsystem.extensions
+
+import com.google.common.truth.Truth.assertThat
+import com.squareup.moshi.Moshi
+import info.metadude.android.eventfahrplan.engelsystem.models.ShiftsResult
+import info.metadude.kotlin.library.engelsystem.EngelsystemService
+import info.metadude.kotlin.library.engelsystem.models.Shift
+import kotlinx.coroutines.runBlocking
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.moshi.MoshiConverterFactory
+import java.io.EOFException
+import java.net.HttpURLConnection
+
+class CallExtensionsTest {
+
+    companion object {
+        private const val VALID_SHIFTS_JSON = "[{\"Comment\":\"This is a very secret comment.\",\"Name\":\"Kirmes\",\"RID\":12,\"SID\":579,\"TID\":10,\"UID\":32,\"created_at_timestamp\":1565979486,\"created_by_user_id\":32,\"description\":\"Kirmes are fun.\",\"edited_at_timestamp\":1567367095,\"edited_by_user_id\":32,\"freeloaded\":0,\"id\":37,\"name\":\"Collect stickers\",\"shifttype_id\":6,\"title\":\"Tag 1: Decorate fridge\"}]"
```

Comment:
Probably looks nicer when using a raw string literal and line breaks.

Reply:
:heavy_check_mark: True. More readable in 0834d19ca714e71a4ebb347a4ccbe518625b3bfe.